.github/workflows/rhobs-release.yaml

```diff
@@ -0,0 +1,76 @@
+# This workflow creates a git tag and publishes container images
+name: rhobs release
+on:
+  push:
+    branches:
+      - 'rhobs-rel-**'
+
+jobs:
+  debug:
+    runs-on: ubuntu-latest
+    if: "!startsWith(github.event.head_commit.message, 'chore(release):')"
+    steps:
+      - name: Debug
+        run: |
+          echo "Skipping release workflow since commit message does match the convention"
+
+  create-release:
+    runs-on: ubuntu-latest
+    if: "startsWith(github.event.head_commit.message, 'chore(release):')"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Import common environment variables
+        run: cat ".github/env" >> $GITHUB_ENV
+
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '${{ env.golang-version }}'
+
+      - name: login to quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.quay_repo_username }}
+          password: ${{ secrets.quay_repo_password }}
+
+      - name: create git tag
+        id: git_tag
+        run: |
+          version="$(head -1 VERSION)"
+          git config user.name rhobs-release-bot
+          git config user.email release-bot@monitoring.rhobs.io
+
+          git tag -a "v${version}" -m "v${version}"
+          git tag -a "pkg/apis/monitoring/v${version}" -m "v${version}"
+          git tag -a "pkg/client/v${version}" -m "v${version}"
+
+
+      - name: Build RHOBS images and push
+        env:
+          IMAGE_ORG: ${{ secrets.IMAGE_ORG }}
+        run: |
+          REGISTRIES="quay.io" \
+          IMAGE_ORG=$IMAGE_ORG \
+          CPU_ARCHS="amd64" \
+          TAG="v$(head -1 VERSION)" \
+          ./rhobs/push-container-images.sh
+
+          echo ::set-output name=tag_name::$(git describe HEAD --abbrev=0)
+
+      - name: setup rhobs module import test
+        run: |
+          cd rhobs/test/import
+          go mod edit -dropreplace github.com/rhobs/obo-prometheus-operator
+          go mod edit -dropreplace github.com/rhobs/obo-prometheus-operator/pkg/client
+          go mod edit -dropreplace github.com/rhobs/obo-prometheus-operator/pkg/apis/monitoring
+          go mod tidy
+          go test ./...
```

Comment:
* Should we move this one step above (soon after the tags are created) so that the images aren't pushed? 
* How does this work if the tags aren't pushed, how does go resolve `github.com/rhobs/obo-prometheus-operator/pkg/apis/monitoring v0.64.0-rhobs2` ?

Reply:
will merge this into previous step
if tags dont get pushed the tests may not work.